### ISSUE TITLE
Track data changes early

### ISSUE THREAD

**Issue by Johnf2021** [erikrichardlarson/unbox#33]
I noticed that as I'm playing the next track on the headphones getting ready for the mix (as you do) even this can trigger the data to update.

So it seems to start that 60 second countdown as soon as I hit play on the track I'm mixing in but really it would be much better if the track data only updated once the cross fader is completely across to the next track. I don't know if this is even possible as I guess your limitations are down to what Pioneer actually offers (I'm using Rekordbox)

**Comment by erikrichardlarson** [comment]
Yeah this is limitation on the rekordbox side when polling their database, are you on windows or mac? If you're on windows you can try the real-time mode, or if you're on windows you can use the KUVO mode, either of these will follow the cross fader / master channel.

**Comment by Johnf2021** [comment]
Ok no problem. Kuvo is not something I can use as it lasts 25 minutes then stops. It does this repeatedly so I have asked them why this is. As it is, its ok because I tend to know the tracks Im mixing so don't need to play too much of them through the headphones and when I just finish the mix, the track changes sd it times quite right actually!

**Comment by Nass86** [comment]
On this topic is there a way for it to follow the "Master" channel?

On Pioneer equipment, once I've moved the crossfader across enough (or upfader up enough) and the last track is faded out, the Master changes.

That could be a great signal for the track rather than using a 60 / 70 / 80 second timer.

**Comment by Johnf2021** [comment]
@Nass86 I have a feeling this is what 'Rekordbox Real-Time' will do. It's an option on the Windows version of the plugin. I can't test yet as I need to wait for an update to the plugin so it works with the version of Rekordbox I have 6.5.1
